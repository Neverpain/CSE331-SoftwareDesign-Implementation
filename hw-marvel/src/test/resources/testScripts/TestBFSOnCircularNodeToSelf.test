# Tests findPath method from a node, to another node, than back to the original
# node and prints out the path

CreateGraph g

AddNode g n1
AddNode g n2

AddEdge g n1 n2 e1
AddEdge g n2 n1 e2

FindPath g n1 n1