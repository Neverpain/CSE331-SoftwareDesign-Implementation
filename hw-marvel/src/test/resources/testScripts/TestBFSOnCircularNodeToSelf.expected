# Tests findPath method from a node, to another node, than back to the original
# node and prints out the path

created graph g

added node n1 to g
added node n2 to g

added edge e1 from n1 to n2 in g
added edge e2 from n2 to n1 in g

path from n1 to n1: